app/components/countdown_component.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class CountdownComponent < BaseComponent
-  attr_reader :expiration, :update_interval, :start_immediately, :tag_options
+  attr_reader :expiration, :update_interval, :start_immediately, :tag_options,
+              :delay_countdown_announcement
```

Comment:
It doesn't look like we're using the `delay_countdown_announcement` attribute?

```suggestion
  attr_reader :expiration, :update_interval, :start_immediately, :tag_options
```

Reply:
Not anymore. It will be removed